docs/tls/README.md

```diff
@@ -8,7 +8,7 @@ In this document, we will configure Minio servers with TLS certificates for both
 
 ## 2. Configure with existing certificates
 
-Assuming that you are already having private and public certificates, you will need to copy them under `certs` in your Minio config directory using the names `private.key` and `public.crt` for key and public certificates respectively.
+Assuming that you are already having private and public certificates, or passwordless PKCS#12 certificate, you will need to copy them under `certs` in your Minio config directory using the names `private.key` and `public.crt` for key and public certificates respectively, or `cert.pfx` in case of PKCS#12 certificate.
```

Comment:
IMO storing password of the certificate for every run is as insecure as passwordless certificate.  Asking password for every run will lead some service functionality breakage eg. restart.  


> Can we detect if a given pfx file requires password?

Its possible to do